src/Illuminate/Foundation/Console/ApiInstallCommand.php

```diff
@@ -100,6 +100,8 @@ protected function uncommentApiRoutesFile()
                 'api: ',
                 $appBootstrapPath,
             );
+        } elseif (str_contains($content, 'api:')) {
```

Comment:
Some people use formatting style where the parameters are aligned like this:

```php
return Application::configure(basePath: dirname(__DIR__))
    ->withRouting(
        web      : __DIR__ . '/../routes/web.php',
        api      : __DIR__ . '/../routes/api.php',
        commands : __DIR__ . '/../routes/console.php',
        health   : '/up',
        apiPrefix: '',
    )
```

Therefore this check might not always be effective. Thinking if there are better ways to do it 🤔

Reply:
I think such case is covered with the warning in else condition `$this->components->warn('Unable to automatically add API route definition to the bootstrap file. API route file should be registered manually.');`